src/test/java/com/tabnine/plugin/HintTests.kt

```diff
@@ -1,89 +0,0 @@
-package com.tabnine.plugin
```

Comment:
y are these tests removed?

Reply:
Because I can't reproduce the settings now that I refactored it. If you know how to solve that, let me know